docs/todos.md

```diff
@@ -44,28 +44,44 @@ todos can be created with optional due dates. These due dates allow for todos to
 
 Due dates can be configured in multiple ways, but all specify integers for `warn` and `error` to signify the number of days from the todo created date to decay the severity.
 
+Due dates can be configured on a per-rule basis as well with the `daysToDecayByRule` option.  See examples below.
```

Comment:
I'd actually love to move this down below in its own section, so we can call it out explicitly. I sort of view this as a layered approach; we do the simple thing first, followed by successively more complex.

Can we give this its own section, outside of enumerating the different ways you can set the configs. Basically, have a section with a 3 # heading, called something like `Configuring daysToDecay by rule`.